File: michielstokman/ai-implementation-fastapi/app/providers/mock_provider.py
def generate_json_from_prompt(prompt: str) -> dict:
    is_meditation = 'guided meditation' in prompt.lower()
    return {
        'title': 'A Gentle Return' if is_meditation else 'The Door I Finally Opened',
        'type': 'meditation' if is_meditation else 'story',
        'language': 'en',
        'growth_area': 'Fear & Freedom',
        'life_phase': 'Recalibrating',
        'content': 'Close your eyes if that feels safe...' if is_meditation else 'I used to think freedom had to be loud...'
    }

File: michielstokman/ai-implementation/src/providers/mockProvider.js
async function generateJsonFromPrompt(prompt) {
  const isMeditation = prompt.toLowerCase().includes("guided meditation");
  const type = isMeditation ? "meditation" : "story";
  return {
    title: isMeditation ? "A Gentle Return" : "The Door I Finally Opened",
    type,
    language: "en",
    growthArea: "Fear & Freedom",
    lifePhase: "Recalibrating",
    content: isMeditation
      ? "Close your eyes if that feels safe... Breathe slowly..."
      : "I used to think freedom had to be loud. Then one quiet morning...",
  };
}

module.exports = { generateJsonFromPrompt };
